nixos/modules/config/fonts/fontconfig.nix

```diff
@@ -67,12 +67,6 @@ let
         <edit mode="append" name="hintstyle">
           <const>${cfg.hinting.style}</const>
         </edit>
-        <edit mode="append" name="antialias">
-          ${fcBool cfg.antialias}
-        </edit>
-        <edit mode="append" name="rgba">
-          <const>${cfg.subpixel.rgba}</const>
-        </edit>
         <edit mode="append" name="lcdfilter">
           <const>lcd${cfg.subpixel.lcdfilter}</const>
```

Comment:
Ugh, looks like `lcdfilter` has been linked [since 2.13.95](https://gitlab.freedesktop.org/fontconfig/fontconfig/-/blame/7861a719616b4b132b9cac089c6c64f47832edb1/conf.d/meson.build#L51):

- https://gitlab.freedesktop.org/fontconfig/fontconfig/-/blob/7861a719616b4b132b9cac089c6c64f47832edb1/conf.d/11-lcdfilter-default.conf
- https://gitlab.freedesktop.org/fontconfig/fontconfig/-/blob/7861a719616b4b132b9cac089c6c64f47832edb1/conf.d/11-lcdfilter-legacy.conf
- https://gitlab.freedesktop.org/fontconfig/fontconfig/-/blob/7861a719616b4b132b9cac089c6c64f47832edb1/conf.d/11-lcdfilter-light.conf

And `hintstyle` [since 2.12.1](https://gitlab.freedesktop.org/fontconfig/fontconfig/-/commits/2.12.1):

- https://gitlab.freedesktop.org/fontconfig/fontconfig/-/blob/7861a719616b4b132b9cac089c6c64f47832edb1/conf.d/10-hinting-full.conf
- https://gitlab.freedesktop.org/fontconfig/fontconfig/-/blob/7861a719616b4b132b9cac089c6c64f47832edb1/conf.d/10-hinting-medium.conf
- https://gitlab.freedesktop.org/fontconfig/fontconfig/-/blob/7861a719616b4b132b9cac089c6c64f47832edb1/conf.d/10-hinting-none.conf
- https://gitlab.freedesktop.org/fontconfig/fontconfig/-/blob/7861a719616b4b132b9cac089c6c64f47832edb1/conf.d/10-hinting-slight.conf

Reply:
Hmm. The latter will require a breaking change, as the files are named `10-hinting-{full,medium,none,slight}.conf` while the enum for `hinting.style` has the following values `[ "hintnone" "hintslight" "hintmedium" "hintfull" ]`

Alternatively, we could create a substring of these values to avoid a breaking change, which would be less clean in my opinion.

Reply:
Yep. That apply function seems like it's a nicer solution.